lib/prompts/createProjectPrompt.ts

```diff
@@ -85,28 +28,20 @@ function findTemplate(
 }
 
 export async function createProjectPrompt(
```

Comment:
The behavior of this util was a little confusing because it was doing a lot. I moved the template fetch out of it to simplify things a little. Now all this does is prompt for name, dest, and template if a list of templates is provided.